src/options/i18n/locales/de.json5

```diff
@@ -15,19 +15,18 @@
     },
     // Each key in 'groups' is a group defined at https://www.hololive.tv/member and the according value is the localized name of it
     groups: {
+      // This group collects the members who are newly tracked by the backend but have not been updated yet in this extension
+      newDebuts: "Neue Debüts!",
       hololive: "Hololive",
       holostars: "Holostars",
       inonakaMusic: "INoNaKa MUSIC",
       hololiveChina: "Hololive China",
       hololiveIndonesia: "Hololive Indonesia",
-      hololiveEnglish: "Hololive English",
-      others: "Andere"
+      hololiveEnglish: "Hololive English"
     }
```

Comment:
Group 'Others' is renamed to 'New Debuts!'.

I used Google Translation to translate the word 'New Debuts' into `German` and `Japanese` but I am not sure if this expression is weird among native speakers. For example, is it better to use 'Recent' than 'New', or better to use 'joined'/'added' than 'debut' according to native language preferences?

Would you please spare some time and take a look on these translations? @samdotxml @is984 @ReiRokusanami0010 @taizan-hokuto (Since you have contributed to this project recently.)

I will leave this PR open for at least 48 hours. Feel free to ignore me if you are busy or there is no problem. Thank you!

Reply:
This comment for Translation of Japanese

In this case, I think it's more like "recently debuted member".
If you want to label the group as such, you can use "新規メンバー" , "新メンバー","新規デビュー",etc.
There is also the expression "最近デビューしたメンバー," although it is a bit longer.
There are so many different expressions that even Japanese people have a hard time judging them.
In my opinion, "新メンバー" is more appropriate, but it is difficult to say for sure because of the reader's preference.
I'm sorry I don't have a clear answer for you.

Note: The phrase "～の (meaning: of)" is rarely used as a proper noun for a group.

Reply:
> This comment for Translation of Japanese
> 
> In this case, I think it's more like "recently debuted member".
> If you want to label the group as such, you can use "新規メンバー" , "新メンバー","新規デビュー",etc.
> There is also the expression "最近デビューしたメンバー," although it is a bit longer.
> There are so many different expressions that even Japanese people have a hard time judging them.
> In my opinion, "新メンバー" is more appropriate, but it is difficult to say for sure because of the reader's preference.
> I'm sorry I don't have a clear answer for you.
> 
> Note: The phrase "～の (meaning: of)" is rarely used as a proper noun for a group.
> @YunzheZJU

Yes, you are right. I am meaning "recently debuted member" here.

Your suggestion is useful! I will take them into consideration.

For me, I am interested in `新規デビュー` and `新メンバー`. They are short and meaningful enough.

You've provided so much information. Thank you!